### PR TITLE
fix: project settings fields

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
Fix project setting field.

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [x] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Project Settings Timezone Error](https://trello.com/c/I5yQcTa9/784-project-settings-timezone-error)

**Comment by tristanhcole** [comment]
[Project Settings Timezone Error](https://trello.com/c/I5yQcTa9/784-project-settings-timezone-error)

File: app/migrations/versions/3b14e2aff4ea_.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# revision identifiers, used by Alembic.
+revision = '3b14e2aff4ea'
+down_revision = '7b3486815759'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('organisation', '_timezone',
```
Comment (michielderoos) [inline app/migrations/versions/3b14e2aff4ea_.py:21]: This migration fails on existing databases where there are existing `null` timezones   
image

**Review by michielderoos — CHANGES_REQUESTED** [review]
Just one little note, otherwise looks good!

**Review by michielderoos — APPROVED** [review]
Had to fix the tests, but LGTM!

**Pull request merged**